### ISSUE TITLE
Problem with switch and filters in php7

### ISSUE THREAD

**Issue by tacxticx88** [phalcon/cphalcon#13242]
Hello,

In the new version (3.3) of Phalcon, the switch was brought in php7. My project is in the process of migrating from php5 to php7, so I have just encountered a problem between the switch and the default filter.

![screenshot_20171230_112339](https://user-images.githubusercontent.com/12997062/34453373-e49c7e58-ed53-11e7-9af3-97884b785b28.png)

```php
{{ myvar|default("abc") }}
```

**Comment by sergeyklay** [comment]
Could you please provide the code to reproduce?

**Comment by Jurigag** [comment]
He provided, just use this code in volt.

**Comment by tacxticx88** [comment]
Parts of volt file : 
```
{% for c in columns %}  
   <th data-column-id="{{c['name']}}" data-type="{{c['type']}}" data-sortable="{{c['sortable']}}" data-visible={{ c['visible']|default('true') }}>{{c['label']}}</th>
{% endfor %}
```

**Comment by sergeyklay** [comment]
I'll try to sort out as soon as I can (most likely after 3rd January)

**Comment by sergeyklay** [comment]
Fixed in the `3.3.x` branch. Feel free to open a new issue if the problem appears again. Thank you for contributing.
